src/definitions/swap.ts

```diff
@@ -1 +1,2 @@
 export const ListSwapAssets = 'cdp_listSwapAssets';
+export const GetSwapQuote = 'cdp_getSwapQuote';
```

Comment:
Side point, we might want to use the UPPER_CASE standard for those const, and keep them alphabetical order. So do.

```
export const CDP_GET_SWAP_QUOTE = 'cdp_getSwapQuote';
export const CDP_LIST_SWAP_ASSETS = 'cdp_listSwapAssets';
```